_includes/js/getting-started.md

```diff
@@ -46,6 +46,16 @@ Parse.initialize("YOUR_APP_ID", "YOUR_JAVASCRIPT_KEY");
 Parse.serverURL = 'http://YOUR_PARSE_SERVER:1337/parse'
 ```
 
+If the Masterkey needs to be provided, use this
```

Comment:
can you note that the master should only be used in safe environments and never be used on the clientside?

Reply:
sure, will try to add it.

Reply:
Thanks a lot, each time we talk about the master key, we should have a big warning ⚠️ or ‼️ To make sure everyone pays attention :)